src/components/Login/Login.js

```diff
@@ -13,16 +13,8 @@ class Login extends React.Component {
     }
   }
 
-  updateName = (e) => {
```

Comment:
Controlled forms need to have an individual update method per individual input. Please revert this page back to its original state and then I can merge this PR.

Reply:
I had revert the methods in the Login component.